docs/DOCS_README.md

```diff
@@ -4,9 +4,15 @@ The documentation found in this directory is hosted at:
 
 - https://tendermint.com/docs/
 
-and built using [VuePress](https://vuepress.vuejs.org/) from the tendermint website repo:
+and built using [VuePress](https://vuepress.vuejs.org/) like below:
 
-- https://github.com/tendermint/tendermint.com
+```bash
+npm install -g vuepress # global install vuepress tool, only once
+npm install
+
+mkdir -p .vuepress && cp config.js .vuepress/
+vuepress build
+```
 
 Under the hood, Jenkins listens for changes (on develop or master) in ./docs then rebuilds
 either the staging or production site depending on which branch the changes were made.
```

Comment:
Can you add a sentence that says:

To view the latest documentation on the develop branch, see the staging site:

- https://tendermint-staging.interblock.io/docs/

and the documentation on master branch is found here:

- https://tendermint.com/docs/